### PR TITLE
feat: show proposals-archive

### PR DESCRIPTION
**Pull request by tiagoalvesdulce**
This PR implements a way to show the proposals from `proposals-archive` on `proposals`. It will list the proposals in the proposals list and link to the appropriate link in the `proposals-archive` subdomain.

~TODO: create an indicator to clearly show the user which proposals are from `proposals-archive`~
Added a Tooltip to indicate the user that the prop is from the git be.

image

image

### CODE REVIEW

File: src/hooks/api/useLegacyVettedProposals.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import tokenInventory from "src/legacytokeninventory.json";
+
+const mapOldToNewStatus = {
+  // old public
+  4: 2,
+  // old abandoned
+  6: 4
+};
+
+console.log(tokenInventory);
```
Comment (victorgcramos) [inline src/hooks/api/useLegacyVettedProposals.js:14]: Log

File: src/hooks/api/useLegacyVettedProposals.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  [PROPOSAL_VOTING_INELIGIBLE]: "abandoned"
+};
+
+export default function useLegacyVettedProposals(shouldReturn, status) {
+  const [legacyProposals, setLegacyProposals] = useState([]);
+  const [legacyProposalsTokens, setLegacyProposalsTokens] = useState({});
+  useEffect(() => {
+    if (shouldReturn) {
+      const proposalsTokensList = tokenInventory[mapStatusToString[status]];
+      const finalList = newLegacyProposalsInfo.filter(p => proposalsTokensList && proposalsTokensList.includes(p.censorshiprecord.token)).reduce((acc, cur) => ({ ...acc, [cur.censorshiprecord.token]: cur }), {});
```
Comment (victorgcramos) [inline src/hooks/api/useLegacyVettedProposals.js:30]: I think you could move this to a function, or write a comment describing what's going on here

**Review by victorgcramos — COMMENTED** [review]
Code Looks good! Left some inline comments

**Review by lukebp — APPROVED** [review]
tACK

**Pull request merged**